packages/react-hooks/src/useFetch.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+export const useFetch = (url: RequestInfo) => {
+  const [data, setData] = useState(null);
+
+  // empty array as second argument equivalent to componentDidMount
+  useEffect(() => {
+    async function fetchData() {
+      console.log('Url', url);
+      const response = await fetch(url);
+
+      console.log(response);
+      const json = await response.json();
+
+      setData(json);
+    }
+
+    fetchData();
+  }, [url]);
```

Comment:
If the data is fetched after the component is unmounted  `setData` will throw an error as a state of the unmounted component can't be changed.

Also, It will help to use our `LoggerService` for future integrations

```suggestion
useEffect(() => {
    const controller = new AbortController();
    const logger = LoggerService.get('App');

    async function fetchData() {
      try {
        const response = await fetch(url, { signal: controller.signal });
        const json = await response.json();
        logger.trace(`Got date for url ${url}`, json);
        setData(json);
      } catch (e) {
        logger.error(e);
      }
    }

    fetchData();
    // Aborting the Request if the component is unmounted before Resolving
    return () => controller.abort();
  }, [url]);
```